docs/src/pages/AndroidHorizontalNavPage/AndroidHorizontalNavPage.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import readme from '../../../../backpack-android/docs/HorizontalNav/README.md';
+import screenshotDefault from '../../../../backpack-android/docs/HorizontalNav/screenshots/default.png';
+import DocsPageBuilder from '../../components/DocsPageBuilder';
+
+const components = [
+  {
+    id: 'default',
```

Comment:
I would call this something like "All varieties" as we're showing more than the "default" configuration in this screenshot

Reply:
Done